core/ui_mixins/quick_panel.py

```diff
@@ -168,7 +162,7 @@ def __init__(self, items, on_done, **kwargs):
         self.on_done = on_done
         for option in ['flags', 'selected_index', 'on_highlight',
                        'limit', 'next_message', ]:
-            if option in kwargs:
+            if option in kwargs and kwargs[option]:
```

Comment:
It seems that your comment has been discarded. The reason that we need to check the nullness is the usage of LogPanel. LogPanel has a default on_highlight method. If the passing argument is not checked for beingNone, the None value would override the default.